src/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+module.exports = {
+  Relay: require('./circuit/relay'),
+  Dialer: require('./dialer'),
+  multicodec: require('./multicodec'),
+  tag: 'Circuit'
+}
```

Comment:
I believe there should be here a file for

- Dialer - what you currently have
- Hop - the handler for the protocol `/libp2p/relay/hop`
- Stop - the handler for the protocol `/libp2p/relay/stop`